install/README.md

```diff
@@ -0,0 +1,23 @@
+# Quick install scripts
+
+This directory contains scripts for installing Babelfish for PostgreSQL. 
+
+By running the install.sh script it will perform the installation process for your linux distribution. 
+
+Currently the supported linux distribution for this script are:
+
+- Centos 7
+- Centos 8
+- Ubuntu 20.04
+- Debian bullseye
+- Fedora 36
+- Amazon Linux 2
+
+## Install instructions configurations
+The are some environment variable that can be set in order to customize the installation script. 
```

Comment:
How about:

You can use environment variables to customize the installation script: